src/routes/embedded/support/unpartitioned-state/unpartitioned-state.view.tsx

```diff
@@ -0,0 +1,289 @@
+import inAppScreenshotSrc from "url:assets/screenshots/unpartitioned-state/raw-in-app-dark.png";
+
+import styles from "./unpartitioned-state.module.scss";
+
+const pretendToBeBrave = false;
+const isBrave = pretendToBeBrave || window.navigator.brave;
+
+const pretendToBeMobileChrome = false;
+const isMobileChrome = pretendToBeMobileChrome || false;
+
+const pretendToBeInAppAndroidBrowser = false;
+const isInAppAndroidBrowser =
+  pretendToBeInAppAndroidBrowser ||
+  (navigator.userAgent.includes("Android") &&
+    (navigator.userAgent.includes("wv") || navigator.userAgent.includes("WebView")));
+
+// Note this flags are only useful to see the UI in those browsers, but not to reproduce their exact behavior.
+const isPretendingToBeAnotherBrowser = pretendToBeBrave || pretendToBeMobileChrome || pretendToBeInAppAndroidBrowser;
+
+export function UnpartitionedStateMissingEmbeddedView() {
+  const { navigate } = useLocation();
+  const { authStatus, unpartitionedStateStatus, unpartitionedStateConfirmed, confirmUnpartitionedState } =
+    useEmbedded();
+
+  const couldProbablyGetAccess =
+    unpartitionedStateStatus === "rejected" || unpartitionedStateStatus === "error" || isPretendingToBeAnotherBrowser;
+  const shouldTryToGetAccess = authStatus === "noAuth" && couldProbablyGetAccess;
+
+  // Loading state:
+
+  const [isConfirming, setIsConfirming] = useState(false);
+  const [isRequestingPermission, setIsRequestingPermission] = useState(false);
+
+  const isViewLoading =
+    authStatus === "unknown" || authStatus === "loading" || authStatus === "authLoading" || isConfirming;
+  const areButtonsDisabled = isViewLoading || isRequestingPermission;
+
+  // Checkbox and error handling:
+
+  const [isChecked, setIsChecked] = useState(false);
+  const [errorsWhileRequestingAccess, setErrorsWhileRequestingAccess] = useState(0);
+
+  // Handlers:
+
+  const handleContinueAnyway = useCallback(async () => {
+    try {
+      setIsConfirming(true);
+      await confirmUnpartitionedState(isChecked);
+      navigate(EmbeddedPaths.Auth);
+    } catch (error) {
+      toast.error("Unexpected error. Please, try again.");
+      setIsConfirming(false);
+    }
+  }, [confirmUnpartitionedState, isChecked, navigate]);
+
+  const handleRequestPermission = useCallback(async () => {
+    try {
+      setIsRequestingPermission(true);
+
+      const localStorage = await LocalStorage.getInstance();
+
+      if (["rejected", "error"].includes(localStorage.status))
+        throw new Error(`Unpartitioned state status = "${localStorage.status}"`);
+
+      navigate(EmbeddedPaths.Auth);
+    } catch (error) {
+      // Brave throws a "NotAllowedError" error while trying to request access, even after a user interaction...
+      toast.error("Unexpected error. Please, try again.");
+      setErrorsWhileRequestingAccess((prevErrorsWhileRequestingAccess) => prevErrorsWhileRequestingAccess + 1);
+      setIsRequestingPermission(false);
+    }
+  }, [navigate]);
+
+  useInterval(
```

Comment:
With this, the user doesn't need to click the button. Still, the button is there just in case and to trigger the "Continue anyway" flow if for whatever reason we cannot get access.